spec/dry/auto_inject_spec.rb

```diff
@@ -558,5 +578,13 @@ def initialize(other)
         end
       end
     end
+
+    context 'autoinject in a module' do
+      it 'raises exception' do
+        expect {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using `{...}` for multi-line blocks.](https://app.codacy.com/gh/dry-rb/dry-auto_inject/pullRequest?prid=4763762)